client/jetpack-onboarding/steps/summary.jsx

```diff
@@ -19,7 +21,20 @@ class JetpackOnboardingSummaryStep extends React.PureComponent {
 			'You enabled Jetpack and unlocked dozens of website-bolstering features. Continue preparing your site below.'
 		);
 
-		return <FormattedHeader headerText={ headerText } subHeaderText={ subHeaderText } />;
+		// TODO: adapt when we have more info
+		const buttonRedirectHref = '#';
+
+		return (
+			<Fragment>
+				<DocumentHead title={ translate( 'Summary ‹ Jetpack Onboarding' ) } />
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Homepage ‹ Jetpack Onboarding' )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/jetpack-onboarding-summary-site)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).